posts/taking-learning-outside-the-classroom-into-the-cloud.md

```diff
@@ -0,0 +1,54 @@
+
+
+<h2>Resources and Opportunities with Ippon</h2>
+
+If you’ve ever researched how to learn a new skill efficiently, you may already know that the best way is to practice every day. Ipponites are encouraged to invest a little bit of time every day to learn something new, and it’s almost impossible to not be inspired by new curiosities with all the resources and opportunities that Ippon makes available. As an intern, CLIPs (short learning modules) helped me accelerate my learning, especially around testing Java microservices. Even as a full time engineer, I still come back to them, but by far my favorite opportunity to learn has been *Ippon Day*: a full day ‘conference’ dedicated to presentations and workshops hosted by other Ipponites. I had the opportunity to demo integrating Amazon Alexa with an external API, and I especially enjoyed just listening to colleagues talk so passionately about their personal projects, various interests, and learnings from them.
+
+<h2>Working From Home </h2>
+
+While I (and everyone else working from home) have been extremely privileged to work from home for the past year, it has also presented its own set of unforeseen challenges. For me, focus (or lack thereof) became a daily obstacle, which became difficult to decipher from general burnout. And what became particularly difficult was the accompanying imposter syndrome, the feeling of not seeing oneself as ‘fit for the job,’ when in reality we all owe ourselves so much more grace than that. Contextualizing our circumstances has become so important to maintaining self-care during the pandemic, especially when it comes to our definitions of productivity and worthiness. If there’s one lesson we can all take away from this experience of working remotely, I hope it is this idea of grace and reflecting internally on how we measure success and sense of worth.
+
+
+<h1>Looking Ahead</h1>
+
+Taking time off from school to gain experience in a professional work environment has taught me so much more than I was expecting to learn, not only in my technical skills but also in my time management, workflow, and goals for the future. These past nine months have given me the chance to pause and reflect on what I’ve accomplished in college so far, and what I want to accomplish in my last year with the experiential context of what life can look like after college.
+
+As I look ahead to the next year, I know I will miss all of the amazing resources, knowledge sharing, and engaging work that defined my experience with Ippon. Definitely what I will miss most, however, is not a what, but a who: The folks at Ippon are a special community of people who jump at the opportunity to uplift one another because they care deeply about people, not just work. I’m immensely grateful to have been a part of this community and to have benefited from the graciousness of so many talented people.
```

Comment:
```suggestion
As I look ahead to the next year, I know I will miss all the amazing resources, knowledge sharing, and engaging work that defined my experience with Ippon. What I will miss most, however, is not a what, but a who: The folks at Ippon are a special community of people who jump at the opportunity to uplift one another because they care deeply about people, not just work. I’m immensely grateful to have been a part of this community and to have benefited from the graciousness of so many talented people.
```